modules/disk-uploader/pkg/vmexport/vmexport_test.go

```diff
@@ -0,0 +1,192 @@
+package vmexport_test
+
+import (
+	"context"
+	"os"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	fakek8sclient "k8s.io/client-go/kubernetes/fake"
+
+	v1beta1 "kubevirt.io/api/export/v1beta1"
+	kubevirtfake "kubevirt.io/client-go/generated/kubevirt/clientset/versioned/fake"
+	"kubevirt.io/client-go/kubecli"
+
+	"github.com/golang/mock/gomock"
```

Comment:
Same

Reply:
Also here, I'm not sure how to format it, I have basic golang's formatter executed on this file.